### PR TITLE
[13.0][FIX] dms: Prevent 200 records limit in search panel.

### PR DESCRIPTION
**Pull request by victoralmau**
[FIX] Prevent 200 records limit in search panel.
[FIX] Show a short name in directory kanban view (similar to file view) in search panel.
[IMP] Add title to elements in search panel if text is some large.

Please @pedrobaeza can you review it?

@Tecnativa TT31492

### CODE REVIEW

File: dms/models/base.py [hunk 1]
```diff
@@ -17,3 +18,17 @@ def unlink(self):
             [("res_model", "=", self._name), ("res_id", "in", self.ids)]
         ).unlink()
         return result
+
+    @api.model
+    def search_panel_select_range(self, field_name):
```
Comment (pedrobaeza) [inline dms/models/base.py]: Why not overriding this in the specific models?

Comment (pedrobaeza) [reply dms/models/base.py]: And I prefer that you overwrite by monkey-patching the variable `SEARCH_PANEL_LIMIT` before calling super and restore after than verbatim copying code.

Comment (pedrobaeza) [reply dms/models/base.py]: Please read my second comment to see if feasible.

**Comment by pedrobaeza** [comment]
Why not putting the override in `abstract.dms.mixin`?

File: dms/models/base.py (hunk 1 above)
Comment (victoralmau) [reply dms/models/base.py]: Yes, it is feasible and much more optimal and readable.

**Review by pedrobaeza — APPROVED** [review]

**Comment by victoralmau** [comment]
@Tardo , can you review it?

**Review by etobella — APPROVED** [review]

**Pull request merged**